servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1081,6 +1081,10 @@ void RenderForwardClustered::_fill_render_list(RenderListType p_render_list, con
 					distance = -distance_max;
 				}
 
+				if (p_render_data->cam_ortogonal) {
+					distance = 1.0;
```

Comment:
`1.0` seems like a magic number, but is it really reasonable? Shouldn't it be `0.0` or `distance_min`?

Reply:
A couple of lines down, a call to mesh_surface_get_lod() is called which accepts distance * lod_distance_multiplier which then divides some values by this to get effective screen size. 0.0 would cause the highest LOD to always be shown and distance_min would still take into account the distance between the camera and the mesh which shouldn't happen in an orthogonal camera. 1.0 would then just effectively pass in the lod_distance_multiplier itself which is the size of an orthogonal camera or the fov of a perspective camera.